examples/proxmox-machinedeployment.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  # If you change the namespace/name, you must also
+  # adjust the rbac rules
+  name: machine-controller-proxmox
+  namespace: kube-system
+type: Opaque
+stringData:
+  token: << PM_API_TOKEN >>
+  sshPrivateKey: |
+    -----BEGIN OPENSSH PRIVATE KEY-----
+    ...
+    -----END OPENSSH PRIVATE KEY-----
+---
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: proxmox-machinedeployment
+  namespace: kube-system
+spec:
+  paused: false
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  minReadySeconds: 0
+  selector:
+    matchLabels:
+      foo: bar
+  template:
+    metadata:
+      labels:
+        foo: bar
+    spec:
+      providerSpec:
+        value:
+          sshPublicKeys:
+            - "<< YOUR_PUBLIC_KEY >>"
```

Comment:
nit: would it make sense to use perhaps kustomize/helm or any similar tool to make configurability of the examples a bit easier?

Reply:
Examples are used in e2e tests, so changing this would be necessary across all examples plus the e2e test logic.